src/build/versions-props.groovy

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+//
+// This script generates version.properties, similar to Netty
+//
+
+// Oddly, some properties are not accessible with project object.
+// Need manual configuration to access them, using global properties object.
```

Comment:
Could you elaborate on this? It seems like all properties are being accessed via the 'project' object.